assets/src/components/customers/CustomerDetailsModal.tsx

```diff
@@ -103,6 +104,46 @@ export const CustomerDetailsContent = ({customer}: {customer: any}) => {
           </Paragraph>
         </Box>
       </Flex>
+
+      <Box mb={2}>
+        <Box>
+          <Text strong>Metadata</Text>
+        </Box>
+        {Object.entries(metadata).map(([key, value]: any) => {
+          // Note: the array below contains the date formats that will be captured and formatted by dayjs
+          // This approach is based on the examples found here: https://day.js.org/docs/en/parse/string-format
+          const dateFormatsArr = [
```

Comment:
i think this is an interesting idea, but i'm worried about some confusion here in case this gets parsed incorrectly (e.g. 02/05/2020 could be Feb 5 or May 2), so i'd rather just display whatever string is set in the metadata for now :)

Reply:
Ah, good point! There are always so many things that can go wrong with dates/time 😄 I hadn't worked with Day.js before, so it was fun to play around with it a bit.